pkg/kubelet/kubelet_node_status.go

```diff
@@ -684,7 +684,6 @@ func (kl *Kubelet) markVolumesFromNode(node *v1.Node) {
 // recordNodeStatusEvent records an event of the given type with the given
 // message for the node.
 func (kl *Kubelet) recordNodeStatusEvent(eventType, event string) {
-	klog.V(2).InfoS("Recording event message for node", "node", klog.KRef("", string(kl.nodeName)), "event", event)
```

Comment:
Lacking more context, I'd bump the verbosity to 4 (or more) instead of removing the line.

Reply:
Will `kl.recorder.Eventf(kl.nodeRef, eventType, event, "Node %s status is now: %s", kl.nodeName, event)` have the same information?

Reply:
yes, but events are transient in nature, and I see no harm in having a record in the logs with high enough verbosity level.

On the other hand, if we decide (pending other reviewers' take) to remove the log, then this function becomes a one-liner which can be very much inlined in the call sites.

Reply:
Ok.

Reply:
Just realize events inside kl.recorder.Eventf won't be persistent, depending on etcd event TTL.

Also, checked these logs are used for debugging sometimes.

Need to look at it deeper to see why it's spammy.

Reply:
yes, better logs than events please ... events also has the problem that cause lot of damage to the control plane

Reply:
@CatherineF-dev 
```
rep -r recordNodeStatusEvent pkg/kubelet/
pkg/kubelet/kubelet_node_status.go:// recordNodeStatusEvent records an event of the given type with the given
pkg/kubelet/kubelet_node_status.go:func (kl *Kubelet) recordNodeStatusEvent(eventType, event string) {
pkg/kubelet/kubelet_node_status.go:                     kl.recordNodeStatusEvent(v1.EventTypeNormal, events.NodeNotSchedulable)
pkg/kubelet/kubelet_node_status.go:                     kl.recordNodeStatusEvent(v1.EventTypeNormal, events.NodeSchedulable)
pkg/kubelet/kubelet_node_status.go:             nodestatus.MemoryPressureCondition(kl.clock.Now, kl.evictionManager.IsUnderMemoryPressure, kl.recordNodeStatusEvent),
pkg/kubelet/kubelet_node_status.go:             nodestatus.DiskPressureCondition(kl.clock.Now, kl.evictionManager.IsUnderDiskPressure, kl.recordNodeStatusEvent),
pkg/kubelet/kubelet_node_status.go:             nodestatus.PIDPressureCondition(kl.clock.Now, kl.evictionManager.IsUnderPIDPressure, kl.recordNodeStatusEvent),
pkg/kubelet/kubelet_node_status.go:                     kl.containerManager.Status, kl.shutdownManager.ShutdownStatus, kl.recordNodeStatusEvent, kl.supportLocalStorageCapacityIsolation()),
pkg/kubelet/nodestatus/setters.go:      recordEventFunc func(eventType, event string), // typically Kubelet.recordNodeStatusEvent
pkg/kubelet/nodestatus/setters.go:      recordEventFunc func(eventType, event string), // typically Kubelet.recordNodeStatusEvent
pkg/kubelet/nodestatus/setters.go:      recordEventFunc func(eventType, event string), // typically Kubelet.recordNodeStatusEvent
pkg/kubelet/nodestatus/setters.go:      recordEventFunc func(eventType, event string), // typically Kubelet.recordNodeStatusEvent
```

This are called for a lot of things that may not need to log or even raise an event